projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -318,18 +318,73 @@ public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAna
 		$wp_styles  = $current_wp_styles;
 		$wp_scripts = $current_wp_scripts;
 
+		$script_urls = $this->parse_scripts( $scripts );
+		$style_urls  = $this->parse_styles( $styles );
+
+		sort( $script_urls );
+		sort( $style_urls );
+
+		$allowed_block_types = array_merge(
+			$this->get_core_block_types(),
+			self::ALLOWED_PLUGIN_BLOCKS
+		);
+
 		return rest_ensure_response(
 			array(
-				'allowed_block_types' => array_merge(
-					$this->get_core_block_types(),
-					self::ALLOWED_PLUGIN_BLOCKS
-				),
-				'scripts'             => $scripts,
-				'styles'              => $styles,
+				'allowed_block_types' => $allowed_block_types,
+				'hash'                => hash( 'sha256', implode( $allowed_block_types ) . implode( $script_urls ) . implode( $style_urls ) ),
+				'styles_html'         => $styles,
+				'scripts_html'        => $scripts,
+				'scripts'             => $script_urls,
+				'styles'              => $style_urls,
```

Comment:
I want to highlight that https://github.com/wordpress-mobile/GutenbergKit/pull/148 uses the existing structure, not [the one proposed by Jeremy](https://github.com/wordpress-mobile/block-editor-assets-endpoint/pull/1).

The existing JSON response content is sufficient for https://github.com/wordpress-mobile/GutenbergKit/pull/148 to work.